data/selectors.js

```diff
@@ -19,6 +19,7 @@ const elements = {
   "spanDown": ".ant-input-number-handler-down",
   "ageError": ".ant-form-item-explain-error",
```

Comment:
Maybe we could combine two selectors into one? I already have that selector on line 17 but I want to do some refactoring, so it will be `.ant-form-item-explain-error`.
```suggestion
  "error": ".ant-form-item-explain-error",
```

Reply:
Done